core/kernel/user_ta.c

```diff
@@ -49,12 +49,19 @@
 #include <utee_defines.h>
 #include <util.h>
 
-static void init_utee_param(struct utee_params *up,
+static TEE_Result init_utee_param(struct utee_params *up,
 			const struct tee_ta_param *p, void *va[TEE_NUM_PARAMS])
```

Comment:
Please align with the `(` on the line above.

Reply:
updated